node/resolvers/checkout/index.ts

```diff
@@ -194,13 +194,21 @@ export const mutations: Record<string, Resolver> = {
     if (shouldUpdateMarketingData(marketingData, sessionData)) {
       const newMarketingData = {
         ...(marketingData || {}),
-        utmCampaign: path(['utmParams', 'campaign'], sessionData),
-        utmMedium: path(['utmParams', 'medium'], sessionData),
-        utmSource: path(['utmParams', 'source'], sessionData),
-        utmiCampaign: path(['utmiParams', 'campaign'], sessionData),
-        utmiPart: path(['utmiParams', 'part'], sessionData),
-        utmipage: path(['utmiParams', 'page'], sessionData),
       }
+
+      if (sessionData && Object.keys(sessionData).length > 0) {
```

Comment:
Just checking if session has something in it so we don't override marketingData fields if session request failed